configure.ac

```diff
@@ -1,12 +1,12 @@
 define([sockperf_version], esyscmd([sh -c "head -1 ./build/current-version | tr -d '\n'"]))
 AC_INIT([SOCKPERF], [sockperf_version], [http://github.com/mellanox/sockperf/issues])
 
-AC_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
```

Comment:
It will be good to add `AC_PREREQ` macro to assert that autoconf version is fresh enough.